### PR TITLE
LP-2969 - Fix webinar registrations on email update

### PR DESCRIPTION
**Pull request by aightmunam**
[LP-2969](https://philanthropyu.atlassian.net/browse/LP-2969)

***

#### Description:
- Localize webinar display dates to Arabic when the platform language is Arabic
- Fix webinar registrations issue when the user email is updated

***

## Things to Cross-check:
- [ ] [Core Changes Wiki](https://github.com/OmnipreneurshipAcademy/edx-platform/wiki/Core-Changes) updated if needed?

### CODE REVIEW

**Comment by omnipreneurshipA** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/3512/workflows/workflows_id/jobs/3512/artifacts)**. Coverage !!! :rocket:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/lms/helpers.py (100%)
openedx/adg/lms/our_team/models.py (100%)
openedx/adg/lms/our_team/tests/test_helpers.py (100%)
openedx/adg/lms/tests/test_helpers.py (100%)
openedx/adg/lms/webinars/forms.py (93.3%): Missing lines 62
openedx/adg/lms/webinars/helpers.py (100%)
openedx/adg/lms/webinars/models.py (50.0%): Missing lines 439,448
openedx/adg/lms/webinars/tests/constants.py (100%)
openedx/adg/lms/webinars/tests/test_forms.py (100%)
openedx/adg/lms/webinars/tests/test_helpers.py (100%)
-------------
Total:   105 lines
Missing: 3 lines
Coverage: 97%
-------------
```

**Comment by omnipreneurshipA** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/3512/workflows/workflows_id/jobs/3512/artifacts)**. Coverage !!! :rocket:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/lms/helpers.py (100%)
openedx/adg/lms/our_team/models.py (100%)
openedx/adg/lms/our_team/tests/test_helpers.py (100%)
openedx/adg/lms/tests/test_helpers.py (100%)
openedx/adg/lms/webinars/forms.py (93.3%): Missing lines 62
openedx/adg/lms/webinars/helpers.py (100%)
openedx/adg/lms/webinars/models.py (50.0%): Missing lines 439,448
openedx/adg/lms/webinars/tests/constants.py (100%)
openedx/adg/lms/webinars/tests/test_forms.py (100%)
openedx/adg/lms/webinars/tests/test_helpers.py (100%)
-------------
Total:   105 lines
Missing: 3 lines
Coverage: 97%
-------------
```

**Comment by omnipreneurshipA** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/3512/workflows/workflows_id/jobs/3512/artifacts)**. Coverage !!! :rocket:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/lms/helpers.py (100%)
openedx/adg/lms/our_team/models.py (100%)
openedx/adg/lms/our_team/tests/test_helpers.py (100%)
openedx/adg/lms/tests/test_helpers.py (100%)
openedx/adg/lms/webinars/forms.py (93.3%): Missing lines 62
openedx/adg/lms/webinars/helpers.py (100%)
openedx/adg/lms/webinars/models.py (50.0%): Missing lines 439,448
openedx/adg/lms/webinars/tests/constants.py (100%)
openedx/adg/lms/webinars/tests/test_forms.py (100%)
openedx/adg/lms/webinars/tests/test_helpers.py (100%)
-------------
Total:   105 lines
Missing: 3 lines
Coverage: 97%
-------------
```

**Comment by omnipreneurshipA** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/3512/workflows/workflows_id/jobs/3512/artifacts)**. Coverage !!! :rocket:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/lms/helpers.py (100%)
openedx/adg/lms/our_team/models.py (100%)
openedx/adg/lms/our_team/tests/test_helpers.py (100%)
openedx/adg/lms/tests/test_helpers.py (100%)
openedx/adg/lms/webinars/forms.py (93.3%): Missing lines 62
openedx/adg/lms/webinars/helpers.py (100%)
openedx/adg/lms/webinars/models.py (50.0%): Missing lines 439,448
openedx/adg/lms/webinars/tests/constants.py (100%)
openedx/adg/lms/webinars/tests/test_forms.py (100%)
openedx/adg/lms/webinars/tests/test_helpers.py (100%)
-------------
Total:   105 lines
Missing: 3 lines
Coverage: 97%
-------------
```

**Comment by omnipreneurshipA** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/3512/workflows/workflows_id/jobs/3512/artifacts)**. Coverage !!! :rocket:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/lms/helpers.py (100%)
openedx/adg/lms/our_team/models.py (100%)
openedx/adg/lms/our_team/tests/test_helpers.py (100%)
openedx/adg/lms/tests/test_helpers.py (100%)
openedx/adg/lms/webinars/forms.py (93.3%): Missing lines 62
openedx/adg/lms/webinars/helpers.py (100%)
openedx/adg/lms/webinars/models.py (50.0%): Missing lines 439,448
openedx/adg/lms/webinars/tests/constants.py (100%)
openedx/adg/lms/webinars/tests/test_forms.py (100%)
openedx/adg/lms/webinars/tests/test_helpers.py (100%)
-------------
Total:   105 lines
Missing: 3 lines
Coverage: 97%
-------------
```

**Comment by omnipreneurshipA** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/3512/workflows/workflows_id/jobs/3512/artifacts)**. Coverage !!! :rocket:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/lms/helpers.py (100%)
openedx/adg/lms/our_team/models.py (100%)
openedx/adg/lms/our_team/tests/test_helpers.py (100%)
openedx/adg/lms/tests/test_helpers.py (100%)
openedx/adg/lms/webinars/forms.py (93.3%): Missing lines 62
openedx/adg/lms/webinars/helpers.py (100%)
openedx/adg/lms/webinars/models.py (50.0%): Missing lines 439,448
openedx/adg/lms/webinars/tests/constants.py (100%)
openedx/adg/lms/webinars/tests/test_forms.py (100%)
openedx/adg/lms/webinars/tests/test_helpers.py (100%)
-------------
Total:   105 lines
Missing: 3 lines
Coverage: 97%
-------------
```

**Comment by omnipreneurshipA** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/3512/workflows/workflows_id/jobs/3512/artifacts)**. Coverage !!! :rocket:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/lms/helpers.py (100%)
openedx/adg/lms/our_team/models.py (100%)
openedx/adg/lms/our_team/tests/test_helpers.py (100%)
openedx/adg/lms/tests/test_helpers.py (100%)
openedx/adg/lms/webinars/forms.py (93.3%): Missing lines 62
openedx/adg/lms/webinars/helpers.py (100%)
openedx/adg/lms/webinars/models.py (50.0%): Missing lines 439,448
openedx/adg/lms/webinars/tests/constants.py (100%)
openedx/adg/lms/webinars/tests/test_forms.py (100%)
openedx/adg/lms/webinars/tests/test_helpers.py (100%)
-------------
Total:   105 lines
Missing: 3 lines
Coverage: 97%
-------------
```

**Comment by omnipreneurshipA** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/3513/workflows/workflows_id/jobs/3513/artifacts)**. Coverage !!! :rocket:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/lms/helpers.py (100%)
openedx/adg/lms/our_team/models.py (100%)
openedx/adg/lms/our_team/tests/test_helpers.py (100%)
openedx/adg/lms/tests/test_helpers.py (100%)
openedx/adg/lms/webinars/forms.py (93.3%): Missing lines 62
openedx/adg/lms/webinars/helpers.py (100%)
openedx/adg/lms/webinars/models.py (33.3%): Missing lines 439,448
openedx/adg/lms/webinars/tests/constants.py (100%)
openedx/adg/lms/webinars/tests/test_forms.py (100%)
openedx/adg/lms/webinars/tests/test_helpers.py (100%)
-------------
Total:   104 lines
Missing: 3 lines
Coverage: 97%
-------------
```

**Comment by omnipreneurshipA** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/3513/workflows/workflows_id/jobs/3513/artifacts)**. Coverage !!! :rocket:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/lms/helpers.py (100%)
openedx/adg/lms/our_team/models.py (100%)
openedx/adg/lms/our_team/tests/test_helpers.py (100%)
openedx/adg/lms/tests/test_helpers.py (100%)
openedx/adg/lms/webinars/forms.py (93.3%): Missing lines 62
openedx/adg/lms/webinars/helpers.py (100%)
openedx/adg/lms/webinars/models.py (33.3%): Missing lines 439,448
openedx/adg/lms/webinars/tests/constants.py (100%)
openedx/adg/lms/webinars/tests/test_forms.py (100%)
openedx/adg/lms/webinars/tests/test_helpers.py (100%)
-------------
Total:   104 lines
Missing: 3 lines
Coverage: 97%
-------------
```

**Comment by omnipreneurshipA** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/3513/workflows/workflows_id/jobs/3513/artifacts)**. Coverage !!! :rocket:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/lms/helpers.py (100%)
openedx/adg/lms/our_team/models.py (100%)
openedx/adg/lms/our_team/tests/test_helpers.py (100%)
openedx/adg/lms/tests/test_helpers.py (100%)
openedx/adg/lms/webinars/forms.py (93.3%): Missing lines 62
openedx/adg/lms/webinars/helpers.py (100%)
openedx/adg/lms/webinars/models.py (33.3%): Missing lines 439,448
openedx/adg/lms/webinars/tests/constants.py (100%)
openedx/adg/lms/webinars/tests/test_forms.py (100%)
openedx/adg/lms/webinars/tests/test_helpers.py (100%)
-------------
Total:   104 lines
Missing: 3 lines
Coverage: 97%
-------------
```

**Comment by omnipreneurshipA** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/3513/workflows/workflows_id/jobs/3513/artifacts)**. Coverage !!! :rocket:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/lms/helpers.py (100%)
openedx/adg/lms/our_team/models.py (100%)
openedx/adg/lms/our_team/tests/test_helpers.py (100%)
openedx/adg/lms/tests/test_helpers.py (100%)
openedx/adg/lms/webinars/forms.py (93.3%): Missing lines 62
openedx/adg/lms/webinars/helpers.py (100%)
openedx/adg/lms/webinars/models.py (33.3%): Missing lines 439,448
openedx/adg/lms/webinars/tests/constants.py (100%)
openedx/adg/lms/webinars/tests/test_forms.py (100%)
openedx/adg/lms/webinars/tests/test_helpers.py (100%)
-------------
Total:   104 lines
Missing: 3 lines
Coverage: 97%
-------------
```

**Comment by omnipreneurshipA** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/3513/workflows/workflows_id/jobs/3513/artifacts)**. Coverage !!! :rocket:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/lms/helpers.py (100%)
openedx/adg/lms/our_team/models.py (100%)
openedx/adg/lms/our_team/tests/test_helpers.py (100%)
openedx/adg/lms/tests/test_helpers.py (100%)
openedx/adg/lms/webinars/forms.py (93.3%): Missing lines 62
openedx/adg/lms/webinars/helpers.py (100%)
openedx/adg/lms/webinars/models.py (33.3%): Missing lines 439,448
openedx/adg/lms/webinars/tests/constants.py (100%)
openedx/adg/lms/webinars/tests/test_forms.py (100%)
openedx/adg/lms/webinars/tests/test_helpers.py (100%)
-------------
Total:   104 lines
Missing: 3 lines
Coverage: 97%
-------------
```

**Comment by omnipreneurshipA** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/3513/workflows/workflows_id/jobs/3513/artifacts)**. Coverage !!! :rocket:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/lms/helpers.py (100%)
openedx/adg/lms/our_team/models.py (100%)
openedx/adg/lms/our_team/tests/test_helpers.py (100%)
openedx/adg/lms/tests/test_helpers.py (100%)
openedx/adg/lms/webinars/forms.py (93.3%): Missing lines 62
openedx/adg/lms/webinars/helpers.py (100%)
openedx/adg/lms/webinars/models.py (33.3%): Missing lines 439,448
openedx/adg/lms/webinars/tests/constants.py (100%)
openedx/adg/lms/webinars/tests/test_forms.py (100%)
openedx/adg/lms/webinars/tests/test_helpers.py (100%)
-------------
Total:   104 lines
Missing: 3 lines
Coverage: 97%
-------------
```

**Comment by omnipreneurshipA** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/3513/workflows/workflows_id/jobs/3513/artifacts)**. Coverage !!! :rocket:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/lms/helpers.py (100%)
openedx/adg/lms/our_team/models.py (100%)
openedx/adg/lms/our_team/tests/test_helpers.py (100%)
openedx/adg/lms/tests/test_helpers.py (100%)
openedx/adg/lms/webinars/forms.py (93.3%): Missing lines 62
openedx/adg/lms/webinars/helpers.py (100%)
openedx/adg/lms/webinars/models.py (33.3%): Missing lines 439,448
openedx/adg/lms/webinars/tests/constants.py (100%)
openedx/adg/lms/webinars/tests/test_forms.py (100%)
openedx/adg/lms/webinars/tests/test_helpers.py (100%)
-------------
Total:   104 lines
Missing: 3 lines
Coverage: 97%
-------------
```

**Comment by aightmunam** [comment]
The quality check is failing due to a core edx file, please ignore those.

**Review by faizan-philu — APPROVED** [review]

File: openedx/adg/lms/webinars/helpers.py [hunk 1]
```diff
@@ -288,3 +299,81 @@ def has_any_field_changed(item, old_item_data, fields_to_check):
...
+    Args:
+        date_time_object (datetime): Datetime date, time or datetime object
+        output_format (str): Output formatting for the datetime object
+
+    Returns:
+        str: Localized arabic output string in the specified format
+    """
+    date_in_AST = convert_timezone(date_time_object, WEBINAR_DEFAULT_TIME_ZONE)
+
+    with override('ar'):
```
Comment (asadali145) [inline openedx/adg/lms/webinars/helpers.py]: Please use the constant.

File: openedx/adg/lms/webinars/helpers.py [hunk 2]
```diff
@@ -288,3 +299,81 @@ def has_any_field_changed(item, old_item_data, fields_to_check):
...
+        if output_format == WEBINAR_DATE_TIME_FORMAT:
+            return _('{week_day}, {month} {day}, {year} {hours}:{minutes} {time_format} AST').format(
```
Comment (asadali145) [inline openedx/adg/lms/webinars/helpers.py:358]: Do we really need to do _() over the formatting string?

**Comment by omnipreneurshipA** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/3517/workflows/workflows_id/jobs/3517/artifacts)**. Coverage !!! :rocket:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/lms/constants.py (100%)
openedx/adg/lms/helpers.py (100%)
openedx/adg/lms/our_team/helpers.py (100%)
openedx/adg/lms/our_team/models.py (100%)
openedx/adg/lms/our_team/tests/test_helpers.py (100%)
openedx/adg/lms/tests/test_helpers.py (100%)
openedx/adg/lms/webinars/forms.py (93.3%): Missing lines 62
openedx/adg/lms/webinars/helpers.py (100%)
openedx/adg/lms/webinars/models.py (33.3%): Missing lines 439,448
openedx/adg/lms/webinars/tests/constants.py (100%)
openedx/adg/lms/webinars/tests/test_forms.py (100%)
openedx/adg/lms/webinars/tests/test_helpers.py (100%)
-------------
Total:   109 lines
Missing: 3 lines
Coverage: 97%
-------------
```

**Comment by omnipreneurshipA** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/3517/workflows/workflows_id/jobs/3517/artifacts)**. Coverage !!! :rocket:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/lms/constants.py (100%)
openedx/adg/lms/helpers.py (100%)
openedx/adg/lms/our_team/helpers.py (100%)
openedx/adg/lms/our_team/models.py (100%)
openedx/adg/lms/our_team/tests/test_helpers.py (100%)
openedx/adg/lms/tests/test_helpers.py (100%)
openedx/adg/lms/webinars/forms.py (93.3%): Missing lines 62
openedx/adg/lms/webinars/helpers.py (100%)
openedx/adg/lms/webinars/models.py (33.3%): Missing lines 439,448
openedx/adg/lms/webinars/tests/constants.py (100%)
openedx/adg/lms/webinars/tests/test_forms.py (100%)
openedx/adg/lms/webinars/tests/test_helpers.py (100%)
-------------
Total:   109 lines
Missing: 3 lines
Coverage: 97%
-------------
```

**Comment by omnipreneurshipA** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/3517/workflows/workflows_id/jobs/3517/artifacts)**. Coverage !!! :rocket:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/lms/constants.py (100%)
openedx/adg/lms/helpers.py (100%)
openedx/adg/lms/our_team/helpers.py (100%)
openedx/adg/lms/our_team/models.py (100%)
openedx/adg/lms/our_team/tests/test_helpers.py (100%)
openedx/adg/lms/tests/test_helpers.py (100%)
openedx/adg/lms/webinars/forms.py (93.3%): Missing lines 62
openedx/adg/lms/webinars/helpers.py (100%)
openedx/adg/lms/webinars/models.py (33.3%): Missing lines 439,448
openedx/adg/lms/webinars/tests/constants.py (100%)
openedx/adg/lms/webinars/tests/test_forms.py (100%)
openedx/adg/lms/webinars/tests/test_helpers.py (100%)
-------------
Total:   109 lines
Missing: 3 lines
Coverage: 97%
-------------
```

**Comment by omnipreneurshipA** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/3517/workflows/workflows_id/jobs/3517/artifacts)**. Coverage !!! :rocket:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/lms/constants.py (100%)
openedx/adg/lms/helpers.py (100%)
openedx/adg/lms/our_team/helpers.py (100%)
openedx/adg/lms/our_team/models.py (100%)
openedx/adg/lms/our_team/tests/test_helpers.py (100%)
openedx/adg/lms/tests/test_helpers.py (100%)
openedx/adg/lms/webinars/forms.py (93.3%): Missing lines 62
openedx/adg/lms/webinars/helpers.py (100%)
openedx/adg/lms/webinars/models.py (33.3%): Missing lines 439,448
openedx/adg/lms/webinars/tests/constants.py (100%)
openedx/adg/lms/webinars/tests/test_forms.py (100%)
openedx/adg/lms/webinars/tests/test_helpers.py (100%)
-------------
Total:   109 lines
Missing: 3 lines
Coverage: 97%
-------------
```

**Comment by omnipreneurshipA** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/3517/workflows/workflows_id/jobs/3517/artifacts)**. Coverage !!! :rocket:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/lms/constants.py (100%)
openedx/adg/lms/helpers.py (100%)
openedx/adg/lms/our_team/helpers.py (100%)
openedx/adg/lms/our_team/models.py (100%)
openedx/adg/lms/our_team/tests/test_helpers.py (100%)
openedx/adg/lms/tests/test_helpers.py (100%)
openedx/adg/lms/webinars/forms.py (93.3%): Missing lines 62
openedx/adg/lms/webinars/helpers.py (100%)
openedx/adg/lms/webinars/models.py (33.3%): Missing lines 439,448
openedx/adg/lms/webinars/tests/constants.py (100%)
openedx/adg/lms/webinars/tests/test_forms.py (100%)
openedx/adg/lms/webinars/tests/test_helpers.py (100%)
-------------
Total:   109 lines
Missing: 3 lines
Coverage: 97%
-------------
```

**Comment by omnipreneurshipA** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/3517/workflows/workflows_id/jobs/3517/artifacts)**. Coverage !!! :rocket:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/lms/constants.py (100%)
openedx/adg/lms/helpers.py (100%)
openedx/adg/lms/our_team/helpers.py (100%)
openedx/adg/lms/our_team/models.py (100%)
openedx/adg/lms/our_team/tests/test_helpers.py (100%)
openedx/adg/lms/tests/test_helpers.py (100%)
openedx/adg/lms/webinars/forms.py (93.3%): Missing lines 62
openedx/adg/lms/webinars/helpers.py (100%)
openedx/adg/lms/webinars/models.py (33.3%): Missing lines 439,448
openedx/adg/lms/webinars/tests/constants.py (100%)
openedx/adg/lms/webinars/tests/test_forms.py (100%)
openedx/adg/lms/webinars/tests/test_helpers.py (100%)
-------------
Total:   109 lines
Missing: 3 lines
Coverage: 97%
-------------
```

**Comment by omnipreneurshipA** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/3517/workflows/workflows_id/jobs/3517/artifacts)**. Coverage !!! :rocket:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/lms/constants.py (100%)
openedx/adg/lms/helpers.py (100%)
openedx/adg/lms/our_team/helpers.py (100%)
openedx/adg/lms/our_team/models.py (100%)
openedx/adg/lms/our_team/tests/test_helpers.py (100%)
openedx/adg/lms/tests/test_helpers.py (100%)
openedx/adg/lms/webinars/forms.py (93.3%): Missing lines 62
openedx/adg/lms/webinars/helpers.py (100%)
openedx/adg/lms/webinars/models.py (33.3%): Missing lines 439,448
openedx/adg/lms/webinars/tests/constants.py (100%)
openedx/adg/lms/webinars/tests/test_forms.py (100%)
openedx/adg/lms/webinars/tests/test_helpers.py (100%)
-------------
Total:   109 lines
Missing: 3 lines
Coverage: 97%
-------------
```

**Review by asadali145 — APPROVED** [review]
Awesome Work!

File: openedx/adg/lms/webinars/helpers.py (hunk 2 above)
Comment (aightmunam) [reply openedx/adg/lms/webinars/helpers.py]: Removed these function calls.

**Comment by omnipreneurshipA** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/3519/workflows/workflows_id/jobs/3519/artifacts)**. Coverage !!! :rocket:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/lms/constants.py (100%)
openedx/adg/lms/helpers.py (100%)
openedx/adg/lms/our_team/helpers.py (100%)
openedx/adg/lms/our_team/models.py (100%)
openedx/adg/lms/our_team/tests/test_helpers.py (100%)
openedx/adg/lms/tests/test_helpers.py (100%)
openedx/adg/lms/webinars/forms.py (93.3%): Missing lines 62
openedx/adg/lms/webinars/helpers.py (100%)
openedx/adg/lms/webinars/models.py (33.3%): Missing lines 439,448
openedx/adg/lms/webinars/tests/constants.py (100%)
openedx/adg/lms/webinars/tests/test_forms.py (100%)
openedx/adg/lms/webinars/tests/test_helpers.py (100%)
-------------
Total:   108 lines
Missing: 3 lines
Coverage: 97%
-------------
```

**Comment by omnipreneurshipA** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/3519/workflows/workflows_id/jobs/3519/artifacts)**. Coverage !!! :rocket:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/lms/constants.py (100%)
openedx/adg/lms/helpers.py (100%)
openedx/adg/lms/our_team/helpers.py (100%)
openedx/adg/lms/our_team/models.py (100%)
openedx/adg/lms/our_team/tests/test_helpers.py (100%)
openedx/adg/lms/tests/test_helpers.py (100%)
openedx/adg/lms/webinars/forms.py (93.3%): Missing lines 62
openedx/adg/lms/webinars/helpers.py (100%)
openedx/adg/lms/webinars/models.py (33.3%): Missing lines 439,448
openedx/adg/lms/webinars/tests/constants.py (100%)
openedx/adg/lms/webinars/tests/test_forms.py (100%)
openedx/adg/lms/webinars/tests/test_helpers.py (100%)
-------------
Total:   108 lines
Missing: 3 lines
Coverage: 97%
-------------
```

**Comment by omnipreneurshipA** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/3519/workflows/workflows_id/jobs/3519/artifacts)**. Coverage !!! :rocket:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/lms/constants.py (100%)
openedx/adg/lms/helpers.py (100%)
openedx/adg/lms/our_team/helpers.py (100%)
openedx/adg/lms/our_team/models.py (100%)
openedx/adg/lms/our_team/tests/test_helpers.py (100%)
openedx/adg/lms/tests/test_helpers.py (100%)
openedx/adg/lms/webinars/forms.py (93.3%): Missing lines 62
openedx/adg/lms/webinars/helpers.py (100%)
openedx/adg/lms/webinars/models.py (33.3%): Missing lines 439,448
openedx/adg/lms/webinars/tests/constants.py (100%)
openedx/adg/lms/webinars/tests/test_forms.py (100%)
openedx/adg/lms/webinars/tests/test_helpers.py (100%)
-------------
Total:   108 lines
Missing: 3 lines
Coverage: 97%
-------------
```

**Comment by omnipreneurshipA** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/3519/workflows/workflows_id/jobs/3519/artifacts)**. Coverage !!! :rocket:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/lms/constants.py (100%)
openedx/adg/lms/helpers.py (100%)
openedx/adg/lms/our_team/helpers.py (100%)
openedx/adg/lms/our_team/models.py (100%)
openedx/adg/lms/our_team/tests/test_helpers.py (100%)
openedx/adg/lms/tests/test_helpers.py (100%)
openedx/adg/lms/webinars/forms.py (93.3%): Missing lines 62
openedx/adg/lms/webinars/helpers.py (100%)
openedx/adg/lms/webinars/models.py (33.3%): Missing lines 439,448
openedx/adg/lms/webinars/tests/constants.py (100%)
openedx/adg/lms/webinars/tests/test_forms.py (100%)
openedx/adg/lms/webinars/tests/test_helpers.py (100%)
-------------
Total:   108 lines
Missing: 3 lines
Coverage: 97%
-------------
```

**Comment by omnipreneurshipA** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/3519/workflows/workflows_id/jobs/3519/artifacts)**. Coverage !!! :rocket:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/lms/constants.py (100%)
openedx/adg/lms/helpers.py (100%)
openedx/adg/lms/our_team/helpers.py (100%)
openedx/adg/lms/our_team/models.py (100%)
openedx/adg/lms/our_team/tests/test_helpers.py (100%)
openedx/adg/lms/tests/test_helpers.py (100%)
openedx/adg/lms/webinars/forms.py (93.3%): Missing lines 62
openedx/adg/lms/webinars/helpers.py (100%)
openedx/adg/lms/webinars/models.py (33.3%): Missing lines 439,448
openedx/adg/lms/webinars/tests/constants.py (100%)
openedx/adg/lms/webinars/tests/test_forms.py (100%)
openedx/adg/lms/webinars/tests/test_helpers.py (100%)
-------------
Total:   108 lines
Missing: 3 lines
Coverage: 97%
-------------
```

**Comment by omnipreneurshipA** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/3519/workflows/workflows_id/jobs/3519/artifacts)**. Coverage !!! :rocket:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/lms/constants.py (100%)
openedx/adg/lms/helpers.py (100%)
openedx/adg/lms/our_team/helpers.py (100%)
openedx/adg/lms/our_team/models.py (100%)
openedx/adg/lms/our_team/tests/test_helpers.py (100%)
openedx/adg/lms/tests/test_helpers.py (100%)
openedx/adg/lms/webinars/forms.py (93.3%): Missing lines 62
openedx/adg/lms/webinars/helpers.py (100%)
openedx/adg/lms/webinars/models.py (33.3%): Missing lines 439,448
openedx/adg/lms/webinars/tests/constants.py (100%)
openedx/adg/lms/webinars/tests/test_forms.py (100%)
openedx/adg/lms/webinars/tests/test_helpers.py (100%)
-------------
Total:   108 lines
Missing: 3 lines
Coverage: 97%
-------------
```

**Comment by omnipreneurshipA** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/3519/workflows/workflows_id/jobs/3519/artifacts)**. Coverage !!! :rocket:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/lms/constants.py (100%)
openedx/adg/lms/helpers.py (100%)
openedx/adg/lms/our_team/helpers.py (100%)
openedx/adg/lms/our_team/models.py (100%)
openedx/adg/lms/our_team/tests/test_helpers.py (100%)
openedx/adg/lms/tests/test_helpers.py (100%)
openedx/adg/lms/webinars/forms.py (93.3%): Missing lines 62
openedx/adg/lms/webinars/helpers.py (100%)
openedx/adg/lms/webinars/models.py (33.3%): Missing lines 439,448
openedx/adg/lms/webinars/tests/constants.py (100%)
openedx/adg/lms/webinars/tests/test_forms.py (100%)
openedx/adg/lms/webinars/tests/test_helpers.py (100%)
-------------
Total:   108 lines
Missing: 3 lines
Coverage: 97%
-------------
```

**Review by asadali145 — APPROVED** [review]

**Review by faizan-philu — APPROVED** [review]

**Pull request merged**